app/controllers/ideas_controller.rb

```diff
@@ -43,7 +43,9 @@ def create
 
   # PATCH/PUT /ideas/1
   def update
+    old_assigned_user_id = @idea.assigned_user_id
     if @idea.update(idea_params)
+      send_assigned_user_email if @idea.assigned_user_id != old_assigned_user_id
```

Comment:
It's possible to check if an attribute changed in a model in rails. In this case it would be `@idea.saved_change_to_assigned_user_id?`

That would be a nicer way than tracking it manually. What do you think about doing it that way?